src/data/fintype/basic.lean

```diff
@@ -841,6 +841,19 @@ lemma univ_option (α : Type*) [fintype α] : (univ : finset (option α)) = inse
   fintype.card (option α) = fintype.card α + 1 :=
 (finset.card_cons _).trans $ congr_arg2 _ (card_map _) rfl
 
+/-- If `option α` is a `fintype` then so is `α` -/
+def fintype_of_option {α : Type*} [fintype (option α)] : fintype α :=
+⟨finset.erase_none (fintype.elems (option α)), λ x, mem_erase_none.mpr (fintype.complete (some x))⟩
+
+/-- If `α` is a `fintype` then any type equivalent to `α` is a `fintype` -/
+def fintype_of_equiv [fintype α] (f : α ≃ β) : fintype β :=
```

Comment:
This definitely exists already

Reply:
Ah, so it does, `of_equiv`.  I couldn't find it with `library_search` or `suggest`.